### DISCUSSION TITLE
invalid state token

### DISCUSSION THREAD

**Discussion by kalmarz** [concourse/concourse#6430] [Help & Support] [answered]
I'm trying to run Concourse in a VM on my local computer, with setting `CONCOURSE_EXTERNAL_URL: http://192.168.68.110:8080`.
Everything else is untouched in `docker-compose.yml`.
The containers come up. I can login with fly and do everything I want. However, after logging into the UI I get this `invalid state token` message in the browser. I tried to compile Concourse from scratch, same result.

Your ideas would be greatly appreciated.

**Comment by taylorsilva** [comment]
As you're logging in I would check your browser's address bar to ensure the URL hasn't changed to something else. I can't think of any other settings that you need to change right now, but maybe that will help you find something. Maybe the IP is changing to `localhost` or `127.0.0.1` during the login flow?

**Comment by kalmarz** [comment, marked as answer]
Thank you for the suggestion. I just checked it again (and it worked!) so managed to track the issue down. The IP doesn't change during the login process. However, I run the setup in a VM and when my laptop goes to sleep the clock in the VM goes off sync and that causes the problem.
